src/forms/CurrentBenefits.js

```diff
@@ -23,45 +25,48 @@ import { allBenefitOrders } from '../programs/allBenefitOrders';
  */
 class CurrentBenefitsContent extends React.Component {
   handleRadioChange = (event, inputProps) => {
-    const benefitName = inputProps.name;
-
-    const route = 'benefits';
-
-    const value = cloneDeep(this.props.current.benefits);
+    const benefitName = inputProps.name,
+          route       = `benefits`,
+          value       = cloneDeep(this.props.current.benefits);
 
     if (inputProps.value) {
       value.push(benefitName);
       value.sort(
         // Make sure benefits are in the correct order
         (a, b) => {
-          const aIndex = this.props.benefits.indexOf(a);
-          const bIndex = this.props.benefits.indexOf(b);
+          // @todo Is this code right? Don't we need `.current` in here?
+          const aIndex = this.props.benefits.indexOf(a),
+                bIndex = this.props.benefits.indexOf(b);
 
           return aIndex - bIndex;
         }
       );
-    }
-    else {
+    } else {
       const index = value.indexOf(benefitName);
 
       if (index >= 0) {
         value.splice(index, 1);
       }
-    }
+    }  // ends if input value
 
-    this.props.updateClientValue(event, { route, value, time: 'current' });
-  };
+    this.props.updateClientValue(event, { route, value, time: `current` });
+  };  // Ends handleRadioChange()
 
   render() {
-    const { current, translations, benefits } = this.props;
+    const {
+      current,
+      translations,
+      benefits,
+    } = this.props;
 
     const components = [];
   
     for (let benefitIndex = 0; benefitIndex < benefits.length; benefitIndex++) {
-      const benefit = benefits[ benefitIndex ];
-
-      const translationKey = `i_has_${benefit}_label`;
+      const benefit        = benefits[ benefitIndex ],
+            translationKey = `i_has_${benefit}_label`;
 
+      // @todo This would have to be handled in every translation situation.
+      // @todo Also, is there a way we can include this in the translation report?
```

Comment:
That seems doable. This is, IMO, one of the weaknesses in the current i18n approach--there doesn't seem to be a way to pass variables to the translation messages (only static placeholders). But that's a whole other discussion!

Reply:
It is! Still an important discussion to have. @todo

Reply:
And thanks for the brainstorming.